test/actions/store.js

```diff
@@ -0,0 +1,75 @@
+import test from 'ava';
+import nock from 'nock';
+
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import { CALL_API } from '../../app/scripts/actions';
+import { requestMiddleware } from '../../app/scripts/middleware/request';
+
+const middlewares = [
+  requestMiddleware,
+  thunk
+];
+const mockStore = configureMockStore(middlewares);
+const store = mockStore({
+  api: {
+    tokens: {
+      token: 'fake-token'
+    }
+  }
+});
+
+test.beforeEach((t) => {
+  t.context.defaultConfig = {
+    json: true,
+    resolveWithFullResponse: true,
+    simple: false
+  };
+
+  t.context.expectedHeaders = {
+    Authorization: 'Bearer fake-token',
+    'Content-Type': 'application/json'
+  };
+
+  store.clearActions();
+});
+
+test.serial('dispatches TYPE_INFLIGHT and TYPE actions for API request action', async (t) => {
+  const stubbedResponse = { test: 'test' };
+  nock('http://localhost:5001')
+    .get('/test-path')
+    .reply(200, stubbedResponse);
+
+  const requestAction = {
+    type: 'TEST',
+    method: 'GET',
+    url: 'http://localhost:5001/test-path'
+  };
+  const actionObj = {
+    [CALL_API]: requestAction
```

Comment:
No qualms or changes needed, just curious as to the reason for `[ ]` as opposed to just `CALL_API`.

Reply:
I think it's just a new syntax from ES6:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names

I also notice that these tests do not return the same thing:

```
var TEST = 'foo';
var test2 = { TEST: 'bar' }
console.log(Object.keys(test2))
```


```
var TEST = 'foo';
var test2 = { [TEST]: 'bar' }
console.log(Object.keys(test2))
```

Only in the second format is the object key actually recognized as the value of the literal (`foo`)